google/ggrc-core
# [5OBJ] GGRC-8530 Move 5OBJ's comments into external comments

# Dependencies

_**This PR should be merged into [GGRC-7200](https://github.com/google/ggrc-core/tree/GGRC-7200) branch which is the common one for all 5OBJ. PRs**_

This PR is `on hold` until the dependencies are merged:

- [x] #10615 (Move Contracts & Policies out into separate tables)
- [x] #10631 (Extend 5OBJ with Q-side fields)
- [x] #10602 (Clean HTML from 5OBJ attributes and comments)

# Issue description

Comments of all 5OBJ should be migrated into external comments.

# Steps to test the changes

1. Run the migration and make sure it pass successfully.
2. Make sure that all Comments related to 5OBJ are migrated into `external_comments` table.

# Solution description

* Add the migration which moves Comments related to 5OBJ from `comments` to `external_comments` table using `migration.utils.external_comments` utilities.

# Sanity checklist


# Migration checklist

# PR Review checklist
